tests/js/server/aql/aql-optimizer-rule-use-index-for-sort.js

```diff
@@ -1109,7 +1084,7 @@ function optimizerRuleTestSuite() {
 
     testSortDescWithFilterSplicedSubquery : function () {
       const query = `FOR i IN [123] RETURN (FOR v IN ${colName} FILTER v.a == i SORT v.b DESC RETURN v)`;
-      const plan = AQL_EXPLAIN(query, {}, {optimizer: {rules: ['+splice-subqueries']}}).plan;
+      const plan = AQL_EXPLAIN(query).plan;
```

Comment:
I think you can delete one of `testSortDescWithFilterSubquery`, `testSortDescWithFilterSplicedSubquery`.

Reply:
Yes :+1: